package.json

```diff
@@ -50,5 +50,394 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "acorn": "^7.4.1",
+    "acorn-jsx": "^5.3.1",
+    "aggregate-error": "^3.1.0",
+    "ajv": "^6.12.6",
+    "ansi-colors": "^4.1.1",
+    "ansi-escapes": "^4.3.2",
+    "ansi-regex": "^5.0.1",
+    "ansi-styles": "^4.3.0",
+    "argparse": "^1.0.10",
+    "array-includes": "^3.1.3",
+    "array-union": "^2.1.0",
+    "array.prototype.flat": "^1.2.4",
+    "array.prototype.flatmap": "^1.2.4",
+    "arrify": "^1.0.1",
+    "asn1": "^0.2.4",
```

Comment:
Your root `package.json` still needs to be restored to its original version, otherwise it'll cause more problems for you down the line.